numpy/polynomial/__init__.py

```diff
@@ -15,26 +15,26 @@
 This package provides *convenience classes* for each of six different kinds
 of polynomials:
 
-         ============    ================
-         **Name**        **Provides**
-         ============    ================
-         Polynomial      Power series
-         Chebyshev       Chebyshev series
-         Legendre        Legendre series
-         Laguerre        Laguerre series
-         Hermite         Hermite series
-         HermiteE        HermiteE series
-         ============    ================
+         ========================    ================
+         **Name**                    **Provides**
+         ========================    ================
+         `~polynomial.Polynomial`    Power series
+         `~chebyshev.Chebyshev`      Chebyshev series
+         `~legendre.Legendre`        Legendre series
+         `~laguerre.Laguerre`        Laguerre series
+         `~hermite.Hermite`          Hermite series
+         `~hermite_e.HermiteE`       HermiteE series
+         ========================    ================
 
 These *convenience classes* provide a consistent interface for creating,
 manipulating, and fitting data with polynomials of different bases.
 The convenience classes are the preferred interface for the `~numpy.polynomial`
 package, and are available from the `numpy.polynomial` namespace.
 This eliminates the need to
-navigate to the corresponding submodules, e.g. ``np.polynomial.Polynomial``
-or ``np.polynomial.Chebyshev`` instead of
-``np.polynomial.polynomial.Polynomial`` or 
-``np.polynomial.chebyshev.Chebyshev``, respectively.
+navigate to the corresponding submodules, e.g. ``numpy.polynomial.Polynomial``
+or ``numpy.polynomial.Chebyshev`` instead of
+`numpy.polynomial.polynomial.Polynomial` or 
+`numpy.polynomial.chebyshev.Chebyshev`, respectively.
```

Comment:
And to these ones.

Reply:
Same thing. I can make the link text read `np....` instead of `numpy....`, as before, if you prefer, but I think having them as links is quite handy, especially for readers like me that want all the background info up front.